homeassistant/components/sensor/rainbird.py

```diff
@@ -0,0 +1,81 @@
+"""
+Support for Rain Bird Irrigation system LNK WiFi Module.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/sensor.rainbird/
+"""
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.rainbird import DATA_RAINBIRD
+import homeassistant.helpers.config_validation as cv
+from homeassistant.const import CONF_MONITORED_CONDITIONS
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.helpers.entity import Entity
+
+DEPENDENCIES = ['rainbird']
+
+_LOGGER = logging.getLogger(__name__)
+
+# sensor_type [ description, unit, icon ]
+SENSOR_TYPES = {
+    'rainsensor': ['Rainsensor', None, 'water']
+}
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_MONITORED_CONDITIONS, default=list(SENSOR_TYPES)):
+        vol.All(cv.ensure_list, [vol.In(SENSOR_TYPES)]),
+})
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Set up a sensor for a raincloud device."""
+    controller = hass.data[DATA_RAINBIRD]
+
+    sensors = []
+    for sensor_type in config.get(CONF_MONITORED_CONDITIONS):
+        if sensor_type == 'rainsensor':
```

Comment:
We shouldn't need this `if`. The schema already enforces that `sensor_type` will be in `SENSOR_TYPES`.

Reply:
Removed if